docs/docs/reference/built-in-components/gatsby-slice.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Gatsby Slice API
+---
+
+> Support for the Gatsby Slice API was added in `gatsby@5.0.0`.
+
+To further the improvements seen by [Incremental Builds](https://www.gatsbyjs.com/blog/2020-04-22-announcing-incremental-builds/), Gatsby includes a built-in `<Slice>` component to divide pages into incrementally built "slices".
+
+By using a new `<Slice>` React component in combination with the [`createSlice`](/docs/reference/config-files/actions/#createNodeField) API for common UI features, Gatsby will be able to build and deploy individual pieces of your site that had content changes, not just entire pages.
```

Comment:
I think we can explain the concept better without resorting to quotes or re-using the word itself. It won't click for everyone what `"slices"` means.

So we'll need to describe in simpler words what that "slices" would mean. I don't have a perfect sentence yet but here would be a start:

```suggestion
To further the improvements seen by [Incremental Builds](https://www.gatsbyjs.com/blog/2020-04-22-announcing-incremental-builds/), Gatsby includes a built-in `<Slice>` component that allows you to split pages into individual parts.

By using the `<Slice>` React component in combination with the [`createSlice`](/docs/reference/config-files/actions/#createSlice) API for common UI features, Gatsby will be able to build and deploy individual pieces of your site that had content changes, not just entire pages.
```